app/code/Magento/Catalog/Test/Unit/Model/Product/Attribute/Source/StockStatusTest.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+// @codingStandardsIgnoreFile
```

Comment:
This is not acceptable in 2.3. @sidolov do we have a sanity check for this?

Reply:
@orlangur 
Hi, I have already removed this string.
This code was port from 2.2-develop where as exemple I checked example app/code/Magento/Catalog/Test/Unit/Model/Product/Attribute/Source/StockTest.php

Reply:
I don't see this line removed in this PR.

Reply:
@orlangur
Yes. This PR is forward port from branch 2.2-develop(original branch with fixed issue is 2.2-develop##16209). Thats why this line was here because my class has similar test cases. In branch 2.3-develop it was removed.
Please check commit 004c9bb917b250ae2d6515ec3b96d4099b6f9c2e where it exist.

Reply:
@XxXgeoXxX please check diff: https://github.com/magento/magento2/pull/19077/files

A new test file introduced with such line which is not acceptable. Please remove this line and possible violations, also, please squash all changes into a single commit.

Reply:
Yes, we have static test that verifies codingStandardsIgnoreFile annotation usages

Reply:
@orlangu
Done